docs/Concepts/Multitenancy.md

```diff
@@ -0,0 +1,131 @@
+
+## Architecture
+
+![Multi-tenancy](../images/Multi-tenancy.png)
+
+In this example architecture, an organization represents a tenant with multiple departments, and users within the departments.
+Each tenant operates on its own private state.
+Each user in an organization owns one or more privacy manager key pairs that allow them to operate on their
+organization's private state.
+A network operator administers entitlements and private state access for each organization using the authorization server.
+
+[JSON-RPC security](../HowTo/Use/JSON-RPC-API-Security.md) features are used to manage a user's access to a private state.
+The [authorization server](#enterprise-authorization-server) controls this access.
+
+## Network topology
+
+A network can consist of multi-tenant nodes and single-tenant nodes.
+One or more independent [authorization servers](#enterprise-authorization-server) can be used to protect multi-tenant
+nodes, but one multi-tenant node can only be protected by one authorization server.
+
+## Enterprise authorization server
+
+To support multi-tenancy, you must configure an authorization server, such as [ORY Hydra](https://github.com/ory/hydra)
+or [Okta](https://developer.okta.com/).
+The authorization server can grant private state access to clients via a [private state identifier (PSI)](#private-state-identifier).
+
+### Access token scope
+
+The [JSON-RPC security plugin](../HowTo/Use/JSON-RPC-API-Security.md) enables the `geth` JSON-RPC API server to be an
+OAuth2-compliant resource server.
+A client must first obtain a pre-authenticated access token from an authorization server, then present the access token
+(using an `Authorization` HTTP request header) when calling an API.
+Calls to the GoQuorum API without an authenticated token are rejected.
+
+The value of the scope encoded in an access token (in case of JWT), or Introspection response (in the case of the OAuth2
+Token Introspection API) contains the [RPC scope](../Reference/Plugins/Security.md#oauth2-scopes) and tenant scope,
+which has the following URL-based syntax:
+
+```text
+"psi://[PSI]?self.eoa=0x0&node.eoa=0x0"
+```
+
+In the syntax, `PSI` is the URL-encoded value of the PSI.
+
+For example, for a client that has access to two private states `PS1` and `PS2`, an authorization server operator sets
+up and grants the following scopes to the client:
+
+```text
+"psi://PS1?self.eoa=0x0&node.eoa"
+"psi://PS2?self.eoa=0x0&node.eoa"
+```
+
+A client presenting an access token containing these scopes has full access (read/write/create) to private contracts on
+private states `PS1` and `PS2` using any Ethereum accounts.
+
+## Multiple private states
+
+Multiple private states (MPS) is a feature that allows a GoQuorum node to manage more than one private state.
+This functionality lays the foundation for multi-tenancy.
+
+You can [upgrade your existing GoQuorum and Tessera nodes to enable MPS](../HowTo/Use/Multitenancy/Migration.md).
```

Comment:
Maybe we should let users know what versions they require? So maybe add something like, "If running GoQuorum <version> or earlier or Tessera <version> and earlier, upgrade to a later version to use the MPS functionality."

Or should we assume they are using the "latest" or "stable" version of the products, in which case we don't mention that they need to upgrade at all?